index.js

```diff
@@ -1,5 +1,6 @@
 const axios = require('axios')
 const pluralize = require('pluralize')
+const _ = require('lodash')
```

Comment:
I do wonder if we use use it and take advantage of it fully, or remove it to save ~22k of space.